heron/healthmgr/src/java/com/twitter/heron/healthmgr/HealthManager.java

```diff
@@ -200,12 +192,20 @@ public static void main(String[] args) throws Exception {
         throw new IllegalArgumentException("Invalid mode: " + getOptionValue(cmd, CliArgs.MODE));
     }
 
+    setupLogging(cmd, config);
 
     LOG.info("Static Heron config loaded successfully ");
     LOG.fine(config.toString());
 
+    // load the default config value and override with any command line values
+    String metricSourceClassName = config.getStringValue(PolicyConfigKey.METRIC_SOURCE_TYPE.key());
+    metricSourceClassName = getOptionValue(cmd, CliArgs.METRIC_SOURCE_TYPE, metricSourceClassName);
+
+    String metricsUrl = config.getStringValue(PolicyConfigKey.METRIC_SOURCE_URL.key());
```

Comment:
If MetricsCacheMgr is the source, MetricsCacheMgr port is not determined in the config yaml. The port in the url has to be passed by command line. I suggest pass the url in command line for MetricsCacheMgr. The heron-executor knows MetricsCacheMgr port.